src/components/Avatar/Avatar.react.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from "react";
+import { Icon } from "../";
+import cn from "classnames";
+import AvatarList from "./AvatarList.react";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +image?: string,
+  +style?: Object,
+  +size?: "" | "sm" | "md" | "lg" | "xl" | "xxl",
```

Comment:
To keep this prop optional, and for it to work in generating the components class names, I've had to put in an empty string as a potential type.
Is there a better/alternative solution?

Reply:
Remove the "", and make the first line be something like `const compSize = size || "";`